utility/src/main/java/org/oppia/android/util/parser/GlideImageLoader.kt

```diff
@@ -80,4 +81,14 @@ class GlideImageLoader @Inject constructor(
       is ImageViewTarget -> into(target.imageView)
     }
   }
+
+  private fun convertImageTransformationToGlideTransofmation(
+    transformations: List<ImageTransformation>
+  ): Array<Transformation<Bitmap>> {
+    return transformations.map {
+      when (it) {
+        ImageTransformation.BLUR -> BlurTransformation(context)
+      }
+    }.toTypedArray()
+  }
```

Comment:
```suggestion
  private fun List<ImageTransformation>.toGlideTransformations(): Array<Transformation<Bitmap>> {
    return map {
      when (it) {
        ImageTransformation.BLUR -> BlurTransformation(context)
      }
    }.toTypedArray()
  }
```

Alternative (optional), Kotlin-idiomatic version. Also note small typo in 'transformation'.

Reply:
Thanks, applied.